packages/cli/src/lib/helpers/validators.ts

```diff
@@ -0,0 +1,58 @@
+import { validate, JSONSchema4 } from "json-schema";
+import { compile } from "json-schema-to-typescript";
+import { writeFile } from "fs";
+
+import ManifestSchema from "./schema.json";
+
+export const manifestValidation = (manifest: object) => {
+  const { valid, errors } = validate(manifest, ManifestSchema as JSONSchema4);
+  if (!valid) {
+    let { property, message } = errors[0];
+
+    /*     
+     We should handle three cases or errors:
+     1- When a non-accepted field is added to the manifest
+     2- When the type of the field it's not expected one
+     3- When a required field it's not sent
+    */
+
+    // First case
+    const objectNotDefined = /is not defined in the schema/.test(message);
+    if (objectNotDefined) {
+      const unacceptedVariable = message.match(
+        new RegExp("The property (.*) is not defined")
+      );
+      throw Error(
+        `Field ${
+          unacceptedVariable![1]
+        } is not accepted in the schema. Please check the accepted fields here: LINK_TO_SCHEMA`
```

Comment:
I was thinking that we should show somewhere the structure accepted for the `web3api.yaml`